.mvn/extensions.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+	<extension>
+		<groupId>org.eclipse.tycho</groupId>
+		<artifactId>tycho-build</artifactId>
+		<version>2.7.1</version>
```

Comment:
If we use maven 3.8.5+ this could use a placeholder now

Reply:
Do you mean the 'CI-friendly' versions?
https://maven.apache.org/maven-ci-friendly.html

Reply:
No but here is an example where I described using placeholders in the `extension.xml`:
https://github.com/eclipse/tycho/blob/master/RELEASE_NOTES.md#enhanced-support-for-maven-ci-friendly-versions

Reply:
Thanks for the hint. That's handy for builds where the 'root' pom and the mvn-extensions are defined in the same repo!
But in this case the `tycho-version` used in the poms is inherited from the platform-parent-pom. The `.mvn/extension.xml` is the only location in this repo that contains the value string of the tycho version. So in this case we would only add extra layer of indirection by reading it from the `.mvn/maven.config` but would not save one specification of the Tycho version. And I don't think we should require to specify the Tycho version on the command-line because this would make local builds more complicated and `.mvn/extension.xml`  is only used for local builds.